test/uints.test.ts

```diff
@@ -166,6 +166,37 @@ describe('uints tests', () => {
     }
   })
 
+  test('new big add 256', async () => {
+    {
+      const a = 1n
+      const b = 2n
+      const result = (await uints.methods.newBigAdd256({ args: { a, b } })).returns
+      expect(result).toStrictEqual({ higher: 0n, lower: 3n })
+    }
+    {
+      const a = MaxU256
+      const b = 2n
+      const result = (await uints.methods.newBigAdd256({ args: { a, b } })).returns
+      expect(result).toStrictEqual({ higher: 1n, lower: 1n })
+    }
+    {
+      const a = MaxU256
+      const b = MaxU256
+      const result = (await uints.methods.newBigAdd256({ args: { a, b } })).returns
+      expect(result).toStrictEqual({ higher: 1n, lower: a - 1n })
+    }
+  })
+
+  test('big add 256 vs new big add 256 comparison', async () => {
+    {
+      const a = MaxU256
+      const b = MaxU256
+      const oldResult = await uints.methods.bigAdd256({ args: { a, b } })
+      const newResult = await uints.methods.newBigAdd256({ args: { a, b } })
+      console.log('big add 256', oldResult.gasUsed, newResult.gasUsed)
```

Comment:
I think assertion that the new methods use less gas will be better than just console logging it. Fix it in all other cases too.
Even if they are using less gas it is worth to consider if we still need previous implementations.